
Trival include fixes - needed for the X server.

  - Rene Rebe <rene@rocklinux.org>

--- ./include/asm-sparc/fbio.h.orig	2004-09-23 00:36:30.119451048 +0200
+++ ./include/asm-sparc/fbio.h	2004-09-23 00:37:16.617382288 +0200
@@ -1,6 +1,9 @@
 #ifndef __LINUX_FBIO_H
 #define __LINUX_FBIO_H
 
+#include <linux/compiler.h>
+#include <linux/fb.h>
+
 /* Constants used for fbio SunOS compatibility */
 /* (C) 1996 Miguel de Icaza */
 
--- ./include/asm-sparc64/fbio.h.orig	2004-09-23 00:36:27.144903248 +0200
+++ ./include/asm-sparc64/fbio.h	2004-09-23 00:37:02.339552848 +0200
@@ -2,6 +2,7 @@
 #define __LINUX_FBIO_H
 
 #include <linux/compiler.h>
+#include <linux/fb.h>
 
 /* Constants used for fbio SunOS compatibility */
 /* (C) 1996 Miguel de Icaza */
